acados/sim/sim_irk_integrator.c

```diff
@@ -311,7 +311,7 @@ int sim_irk_workspace_calculate_size(void *config_, void *dims_, void *opts_)
 
 
 
-static void *sim_irk_workspace_cast(void *config_, void *dims_, void *opts_, void *raw_memory)
+static void *sim_irk_cast_workspace(void *config_, void *dims_, void *opts_, void *raw_memory)
 {
```

Comment:
@zanellia  the naming convention is first the struct and then what to do. Therefore "sim_irk_workspace_cast" was correct. Can you set it back please? Similarly to "ocp_qp_in_memory_calculate_size" (used to be "ocp_qp_in_calculate_memory_size") etc